dev/test_all.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+npm --prefix=../verification/curator-service/api/ run-script test-silent
```

Comment:
why silent?

Reply:
This script is aimed at being just a quick && simple check that I didn't break too many things, I wanted the output to be minimal (no per-case log) but that still prints errors if there are any. Let me know if you want me to drop the silent, it just suited my usecase.

Reply:
gotcha, sg, the rest of us can always add a more verbose one if we want it :D